modules/cache/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "id" {
+  value       = var.cache_store == "memcached" ? module.memcached[0].cluster_id : var.cache_store == "redis" ? module.redis[0].id : "none"
```

Comment:
We don't need the third ternary option for "none" now that the input values are restricted.

Reply:
yeah, right.